hca-rest-soap-connector/voa-rest.js

```diff
@@ -6,12 +6,24 @@ var app = module.exports = loopback();
 
 app.set('restApiRoot', '/api');
 
+// app.set('legacyExplorer', false); TODO should this be set? In production, we don't want exporer.
+
 var ds = loopback.createDataSource('soap',
   {
     connector: require('loopback-connector-soap'),
     remotingEnabled: true,
-    // wsdl: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc?wsdl' // The url to WSDL
-    wsdl: path.join(__dirname, './voa.wsdl')
+    url: 'https://vaww.esrpre-prod.aac.va.gov/voa/voaSvc',
```

Comment:
Thanks Robbie. Even though I made changes to this directory, let's sunset review of those files.